app/src/main/java/com/hover/stax/home/MainActivity.kt

```diff
@@ -226,6 +227,15 @@ class MainActivity : AbstractRequestActivity(), BalancesViewModel.RunBalanceList
         Timber.e("received result. %s", data?.action)
         Timber.e("uuid? %s", data?.extras?.getString("uuid"))
 
+        if(resultCode == RESULT_OK) handlePostSDK(requestCode, resultCode, data)
```

Comment:
this needs to be properly integrated into the when statement below. It should really be in `showPopUpTransactionDetailsIfRequired` and should apply to any transaction, not just transfers.

Reply:
Agreed and effecting this: "properly integrated into the when statement"

I'm not quite sure @davkutalek, if "should apply to any transaction, not just transfers." because:

- Balance most likely would have been initiated from the home screen, not the transfer fragment.